web/src/views/DandisetLandingView/DandisetOwnersDialog.vue

```diff
@@ -94,7 +91,16 @@ const userFormatConversion = (users) => users.map(({
   id: _id, level: _accessLevel, login, name: `${firstName} ${lastName}`,
 }));
 
-const addResult = (users) => users.map((u) => ({ ...u, result: `${u.name} (${u.login})` }));
+// eslint-disable-next-line camelcase
+const girderize = (users) => users.map(({ username, first_name, last_name }) => ({
+  id: username,
+  login: username,
+  username,
+  // eslint-disable-next-line camelcase
+  name: (first_name && last_name) ? `${first_name} ${last_name}` : null,
+}));
```

Comment:
Just an FYI you don't need to ignore the eslint rule here, but it's up to you.

```suggestion
const girderize = (users) => users.map(
  ({ username, first_name: firstName, last_name: lastName }) => ({
    id: username,
    login: username,
    username,
    name: (firstName && lastName) ? `${firstName} ${lastName}` : null,
  }),
);
```